doc/en/user/source/data/database/postgis.rst

```diff
@@ -245,4 +245,106 @@ To insert multi-line text (for use with labeling) remember to use escaped text::
    INSERT INTO place VALUES (ST_GeomFromText('POINT(-71.060316 48.432044)', 4326), E'Westfield\nTower');
 
 
+JsonPointer Function support
+----------------------------
+
+GeoServer is able to translate a jsonPointer function to a query using PostgreSQL support for JSON types. 
+The following are the main characteristics of the implementation:
+
+* the jsonPointer function syntax is like the following: ``jsonPointer(attributeName,'/path/to/json/attribute')``;
```

Comment:
Bullet lists in english are different than in italian, you should start each sentence with a capital and complete the sentence with a period, not a semicolon.

Reply:
fixed